fastlane_core/spec/project_spec.rb

```diff
@@ -296,12 +296,24 @@ def within_a_temp_dir
     end
 
     describe "build_settings() can handle empty lines" do
-      it "SUPPORTED_PLATFORMS should be iphonesimulator iphoneos" do
+      it "SUPPORTED_PLATFORMS should be iphonesimulator iphoneos on XCode >= 8.3" do
         options = { project: "./fastlane_core/spec/fixtures/projects/Example.xcodeproj" }
         @project = FastlaneCore::Project.new(options, xcodebuild_list_silent: true, xcodebuild_suppress_stderr: true)
-        expect(FastlaneCore::Project).to receive(:run_command).with("xcodebuild clean -showBuildSettings -project ./fastlane_core/spec/fixtures/projects/Example.xcodeproj 2> /dev/null", { timeout: 10, retries: 3, print: false }).and_return(File.read("./fastlane_core/spec/fixtures/projects/build_settings_with_toolchains"))
+        allow(FastlaneCore::Helper).to receive(:xcode_atleast?).and_return(true)
+        command = "xcodebuild -showBuildSettings -project ./fastlane_core/spec/fixtures/projects/Example.xcodeproj 2> /dev/null"
+        expect(FastlaneCore::Project).to receive(:run_command).with(command.to_s, { timeout: 10, retries: 3, print: false }).and_return(File.read("./fastlane_core/spec/fixtures/projects/build_settings_with_toolchains"))
         expect(@project.build_settings(key: "SUPPORTED_PLATFORMS")).to eq("iphonesimulator iphoneos")
       end
+      unless FastlaneCore::Helper.xcode_atleast?('8.3')
```

Comment:
Allow is used for Mocking, https://www.relishapp.com/rspec/rspec-mocks/v/2-14/docs/method-stubs  so when I call that method i return true or false